### PR TITLE
{math}[GCC/8.3.0] Normaliz v3.8.4 and deps E-ANTIC, Arb, FLINT, CoCoALib, cddlib, nauty

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

Normaliz v3.8.4 in GCC/8.3.0 and dependencies:
* E-ANTIC v0.1.5
* Arb v2.17.0
* FLINT v2.5.2
* CoCoALib v0.99700
* cddlib v0.94j
* nauty v2.7rc5

This release of Normaliz can move from `gompi` to `GCC` because the developers dropped all dependencies on Boost.

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node125.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/f070b372a6e0fa7543f276f0ec4c67d9 for a full test report.

**Comment by Micket** [comment]
ouch
```
== FAILED: Installation ended unsuccessfully (build directory: /local/EB/build/cddlib/0.94j/GCCcore-8.3.0): build failed (first 300 chars): cmd " make install " exited with exit code 2 and output:
Making install in doc
make[1]: Entering directory `/local/EB/build/cddlib/0.94j/GCCcore-8.3.0/cddlib-0.94j/doc'
make[1]: *** No rule to make target `cddlibman.pdf', needed by `all-am'.  Stop.
make[1]: Leaving directory `/local/EB/build/cddlib/ (took 20 sec)
```

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 4 out of 7 (7 easyconfigs in this PR)
vera-c1 - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/3a0c2b514152869f39ba9e0c1c34722e for a full test report.

**Comment by boegel** [comment]
@Micket Any idea about the underlying cause? Missing dep or so?

**Comment by Micket** [comment]
well it's the documentation, needs latex. Clearly, this isn't working correctly:
```
buildopts = "SUBDIRS='lib-src src'"  # build sources but spare the documentation in latex
```

File: easybuild/easyconfigs/c/cddlib/cddlib-0.94j-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('binutils', '2.32'),
+]
+
+dependencies = [('GMP', '6.1.2')]
+
+preconfigopts = "autoreconf -f -i && "
+
+buildopts = "SUBDIRS='lib-src src'"  # build sources but spare the documentation in latex
```
Comment (Micket) [inline easybuild/easyconfigs/c/cddlib/cddlib-0.94j-GCCcore-8.3.0.eb:26]: Looks like `make install` triggers the build of docs as well. We have installopts right?
```suggestion
buildopts = "SUBDIRS='lib-src src'"  # build sources but spare the documentation in latex
installopts = buildopts
```

**Review by Micket — DISMISSED** [review]

File: easybuild/easyconfigs/c/cddlib/cddlib-0.94j-GCCcore-8.3.0.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/c/cddlib/cddlib-0.94j-GCCcore-8.3.0.eb:26]: @Micket good catch, thanks for looking into it

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node379.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/af72583f0b01ec35e9b6164b47120f6d for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/a8dc8affcd7bdf90d814a7509af35f8e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node381.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b54657be24fb18320c4f70c814ad2c9e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
node127.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/cecfd430a8861f0a7e80849d1245a512 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 5 out of 7 (7 easyconfigs in this PR)
node3404.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/71a989766994beb372bc53eefb786f38 for a full test report.

**Comment by lexming** [comment]
@boegel the issue is that the certificate for `pallini.di.uniroma1.it` has expired. Should I switch the easyconfig of `nauty` to `http`?

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node350.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/d00026e547280fadeabc7c865ebc1a9e for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node350.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/fb0bde78fc0d7ba36a34beb4d2862c55 for a full test report.

**Comment by lexming** [comment]
@boegel the certificates for `nauty` were fixed and this PR is again in working condition. :tada:

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node3545.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/7a39d8d2cf32a3bb2346dcec7da864ef for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node3106.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/16ae0461a7958c821b621cf2e2ee7f5c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @lexming!

**Pull request merged**